icefoganalytics/travel-authorization
# Add Relative Path Support To Api Typescript

# Context

See https://github.com/icefoganalytics/elcc-data-management/pull/18/files

This is a massive quality of life improvement, and makes many code references more readable and flexible.

It lets you stop writing code like
```typescript
import { Expense, Form } from "../../../models"
import { ExpensesPolicy } from "../../../policies"
```

And instead write code like
```typescript
import { Expense, Form } from "@/models"
import { ExpensesPolicy } from "@/policies"
```

# Testing Instructions

1. Boot the database service via `dev up`
2. Boot the api service via `dev up_api`
3. Boot the web service via `dev up_web`
4. Check that the app boots correctly and that you can log in.
5. Check that the production build works as well by following the instructions in the README.
